RootSource.cc

```diff
@@ -38,19 +41,16 @@ RootSource::RootSource(std::string const& iName) :
 			       &delayedReader_);
     branches_.emplace_back(b);
     if(eventAuxiliaryBranchName == dataProducts_.back().name()) {
-      eventAuxReader_ = EventAuxReader(dataProducts_.back().address());
+      eventAuxBranch_ = b;
```

Comment:
This seems like a performance regression. In the previous, the call to TBranch::GetAddress happens just once during construction, but not it is happening each event.

Reply:
So I looked at the implementation of TBranch::GetAddress

https://root.cern.ch/doc/master/TBranch_8h_source.html#l00208

and it is just a simple accessor. Therefore I withdraw my concern.